### PR TITLE
Fix k8s guide to avoid suggesting you need cloud

### PR DESCRIPTION
**Pull request by zzstoatzz**

### CODE REVIEW

File: docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx [hunk 1]
```diff
@@ -336,6 +338,16 @@ worker:
 
 ```
 
+#### Using Prefect Server
+```
+worker:
+  serverApiConfig:
+  apiUrl: <dns or ip address of your Prefect Server instance>
```
Comment (jamiezieziula) [inline docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx:345]: i think this should be invented one

Comment (zzstoatzz) [reply docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx]: updated in [indent](https://github.com/PrefectHQ/prefect/pull/14040/commits/92b4daf35c4c48870df122ccb5f7196302c021a8)

**Review by Abdur-rahmaanJ — APPROVED** [review]
Now super clear that we can also self-host.

Maybe for folks used to the project it's clear, but i scouted here and there the whole day and i could not find out another way that prefect cloud.

Because even the official blogs use prefect cloud when deploying.

File: docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx [hunk 2]
```diff
@@ -336,6 +338,16 @@ worker:
...
 
+#### Using Prefect Server
+```
+worker:
+  apiConfig: server
+  config:
+    workPool: <target workpool name>
+  serverApiConfig:
+    apiUrl: <server URL>
+
```
Comment (jamiezieziula) [inline docs/2.19.x/how-to-guides/work-pools/kubernetes.mdx:349]:
```suggestion
```

**Review by jamiezieziula — APPROVED** [review]

**Pull request merged**